experimental/torch_xla2/torch_xla2/tensor.py

```diff
@@ -313,14 +315,17 @@ class Environment(contextlib.ContextDecorator):
     _prng_key: jax.random.PRNGKey
 
 
-    def __init__(self):
+    def __init__(self, configuration=None):
```

Comment:
Would be the same. However, where those kwargs are stored is the question. So instead of on `self` (env class) storing on config might be simpler for creating new envs.

I want to explore a UX where, people pass around env / create new env with different configs to use in local context and then discard, instead of modifying configs globally (then reset, with or without aid of context managers).